src/python/pants/option/global_options.py

```diff
@@ -65,6 +65,13 @@ def register_bootstrap_options(cls, register):
                       'pants.backend.python',
                       'pants.backend.jvm',
                       'pants.backend.codegen',
```

Comment:
Also, it seems "re-registering" won't cause errors, probably later registering can override previous ones?